spec/controllers/works/default_rails_actions_spec.rb

```diff
@@ -184,7 +184,7 @@ def call_with_params(params)
 
     it "doesn't allow a user to create a work in a series that they don't own" do
       @series = create(:series)
-      work_attributes = attributes_for(:work)
+      work_attributes = attributes_for(:work).except(:posted)
```

Comment:
What happens if we don't exclude `:posted` btw?

Reply:
The tests fail because `posted` isn't one of the parameters allowed in the controller.

```
WorksController
  create
    doesn't allow a user to create a work in a series that they don't own (FAILED - 1)

Failures:

  1) WorksController create doesn't allow a user to create a work in a series that they don't own
     Failure/Error:
       params.require(:work).permit(
         :rating_string, :fandom_string, :relationship_string, :character_string,
         :archive_warning_string, :category_string, :expected_number_of_chapters, :revised_at,
         :freeform_string, :summary, :notes, :endnotes, :collection_names, :recipients, :wip_length,
         :backdate, :language_id, :work_skin_id, :restricted, :anon_commenting_disabled,
         :moderated_commenting_enabled, :title, :pseuds_to_add, :collections_to_add,
         :unrestricted,
         current_user_pseud_ids: [],
         collections_to_remove: [],
         challenge_assignment_ids: [],
     
     ActionController::UnpermittedParameters:
       found unpermitted parameter: :posted
     # ./app/controllers/works_controller.rb:902:in `work_params'
     # ./app/controllers/works_controller.rb:282:in `create'
     # ./spec/controllers/works/default_rails_actions_spec.rb:190:in `block (4 levels) in <top (required)>'
     # ./spec/controllers/works/default_rails_actions_spec.rb:189:in `block (3 levels) in <top (required)>'
```